e2e/browserstack.conf.js

```diff
@@ -2,9 +2,9 @@ const environments = require('./environments.js');
 const glob = require('glob');
 const files = glob.sync('./e2e/features/**/*-test.js');
 const nightwatchConfig = {
-  output_folder: 'e2e/reports',
-  globals_path: 'e2e/globals.js',
-  custom_assertions_path: ['e2e/custom-assertions'],
+  output_folder: './e2e/reports',
+  globals_path: './globals.js',
+  custom_assertions_path: ['./e2e/custom-assertions'],
   src_folders: files,
   selenium: {
     start_process: false,
```

Comment:
I had to add  `check_process_delay: 5000` in the selenium config to get this version to work locally. https://github.com/nightwatchjs/nightwatch/issues/2138
```suggestion
     start_process: false,
     check_process_delay: 5000,
```

Reply:
@Benjaki2 looks good, code commited.